packages/checkbox/theme/material/vaadin-checkbox-styles.js

```diff
@@ -147,9 +147,13 @@ registerStyles(
       left: 2px;
     }
 
+    :host([required]:not([disabled])) [part='required-indicator'] {
+      color: var(--material-secondary-text-color);
+    }
+
     :host([required]) [part='required-indicator']::after {
       content: '*';
-      color: var(--material-secondary-text-color);
+      color: inherit;
```

Comment:
Not sure if this is needed since the color should get inherited by default.

Reply:
Removed.